lib/perl/Genome/Model/Build/ReferenceSequence/Converter.t

```diff
@@ -45,6 +48,33 @@ my $diff = Genome::Sys->diff_file_vs_text($converted_bed_file, $expected_result)
 ok(!$diff, 'converter produced expected result')
     or diag('diff: ' . $diff);
 
+{
+    Sub::Install::reinstall_sub({
+            code => sub { return ['1', 'X'] },
+            into => 'Genome::Model::Build::ReferenceSequence',
+            as => 'chromosome_array_ref',
+        });
+    my $dc_converter = Genome::Model::Build::ReferenceSequence::Converter->create(
+        source_reference_build => $ref_seq_a,
+        destination_reference_build => $ref_seq_c,
+        algorithm => 'drop_extra_contigs',
+        resource => 'convert_chrXX_contigs_to_GL',
+    );
+    my $dc_converted_bed_file = $feature_list->converted_bed_file(reference => $ref_seq_c);
+    ok($dc_converted_bed_file, 'got converted BED file');
+
+    Memoize::flush_cache('Genome::FeatureList::processed_bed_file');
```

Comment:
This line doesn't affect the test in its present location.  It could be moved before the call to `->converted_bed_file` or removed.